site/docs/try/configuration.md

```diff
@@ -73,11 +78,54 @@ When setting `nessie.version.store.type=DYNAMO` which enables DynamoDB as the ve
+| `nessie.version.store.persist.repository-id`                    |                     | `String`  | Sets Nessie repository ID (optional). This ID can be used to distinguish multiple Nessie repositories that reside in the same storage instance. |
+| `nessie.version.store.persist.parents-per-commit`               | `20`                | `int`     | Sets the number of parent-commit-hashes stored in Nessie store.                                                                                 |
+| `nessie.version.store.persist.commit-timeout-millis`            | `5000`              | `int`     | Sets the timeout for CAS-like operations in milliseconds.                                                                                       |
+| `nessie.version.store.persist.commit-retries`                   | `Integer.MAX_VALUE` | `int`     | Sets the maximum retries for CAS-like operations.                                                                                               |
+| `nessie.version.store.persist.retry-initial-sleep-millis-lower` | `5`                 | `int`     | Configures the initial lower-bound sleep time in milliseconds of the exponential backoff when retrying commit operations.                       |
+| `nessie.version.store.persist.retry-initial-sleep-millis-upper` | `25`                | `int`     | Configures the initial upper-bound sleep time in milliseconds of the exponential backoff when retrying commit operations.                       |
+| `nessie.version.store.persist.retry-max-sleep-millis`           | `250`               | `int`     | Configures the max sleep time in milliseconds of the exponential backoff when retrying commit operations.                                       |
+| `nessie.version.store.persist.max-incremental-index-size`       | `50 * 1024`         | `int`     | Maximum serialized size of key indexes stored inside commit objects.                                                                            |
+| `nessie.version.store.persist.max-serialized-index-size`        | `200 * 1024`        | `int`     | Maximum serialized size of key indexes stored as separate objects.                                                                              |
+| `nessie.version.store.persist.max-reference-stripes-per-commit` | `50`                | `int`     | Maximum number of referenced index objects stored inside commit objects.                                                                        |
+| `nessie.version.store.persist.assumed-wall-clock-drift-micros`  | `5_000_000`         | `long`    | Sets the assumed wall-clock drift between multiple Nessie instances, in microseconds.                                                           |
+| `nessie.version.store.persist.namespace-validation`             | `true`              | `boolean` | Whether namespace validation is enabled.                                                                                                        |
```

Comment:
Should we mention that changing this to `false` will violate Nessie spec `2.0.0-beta.1`?

Reply:
addressed